src/frontend/packages/lib_classroom/package.json

```diff
@@ -6,7 +6,7 @@
   "module": "lib/index.es.js",
   "types": "lib/index.d.ts",
   "scripts": {
-    "build": "rollup -c",
+    "build": "rm -rf lib && rollup -c",
```

Comment:
this can not be managed by a rollup plugin ?

Reply:
I pushed with a rollup plugin version (https://www.npmjs.com/package/rollup-plugin-delete).

Reply:
But the result is the same.